src/workflows/airtable/activities.ts

```diff
@@ -88,16 +97,24 @@ export async function updateSoftrCreativeUsers(
   recordId: string,
   record: ISoftrCreativesUser
 ) {
-  const base = (await getConnectionAirTable()).base("appxprwH6zsJbTFyM");
+  const base = (await getConnectionAirTable()).base(process.env.SMART_CONTRACT_AIRTABLE_BASE!);
+
+  //const base = (await getConnectionAirTable()).base("appxprwH6zsJbTFyM");
   /*const base = (await getConnectionAirTable()).base(
     `${process.env.CREATIVE_MASTER_AIRTABLE_BASE}`
   );*/
-
+//console.log("Record",record);
+//console.log("Record id",recordId);
   delete record["Start Date"];
-  delete record["Profile image"];
-  delete record["Dashboard - Backend iFrame"];
-  delete record["ACTIVE?"];
-  return await base("Users").update([
+  //delete record["Dashboard - Backend iFrame"];
+  delete record["ACTIVE? (from Creatives Master)"];
+  delete record["Name"];
+  delete record["Email"];
+  delete record["Magic Link"];
+  //delete record["'Token Address'"];
```

Comment:
This is the tricky part about airtable SDK. If we attempt to write into columns with calculated field we will get unauthorized error. we should make sure only columns that we are updating are going through.

Maybe a better approach is to pick columns we are updating rather than deleting unnecessary ones

Reply:
@wasif1024 any comment on this?

Reply:
This is also resolved